pages/public/index.html

```diff
@@ -33,19 +33,19 @@ <h1>Release - 1.1.1 (Beethoven)</h1>
             This release has even more new features and bugfixes over the 1.0 community release, and should be backwards compatible.
             Please report any bugs <a href="https://github.com/SynthstromAudible/DelugeFirmware/issues/new?assignees=&labels=&projects=&template=bug_report.yml">on github</a>, as you find them.
         </p>
+        <a class="btn" target="_blank" href="https://github.com/SynthstromAudible/DelugeFirmware/releases/download/release_1_1_1/deluge-community-release-1_1_1.zip">Download 1.1.1 (Beethoven)</a>
+        <a class="btn" target="_blank" href="https://github.com/SynthstromAudible/DelugeFirmware/blob/release/1.1/CHANGELOG.md">Changelog current stable</a>
         <a class="btn" target="_blank" href="https://github.com/SynthstromAudible/DelugeFirmware/releases/tag/release_1_1_1">Release notes</a>
-        <a class="btn" target="_blank" href="https://github.com/SynthstromAudible/DelugeFirmware/blob/release/1.1/CHANGELOG.md">Changelog</a>
-        <a class="btn" target="_blank" href="https://github.com/SynthstromAudible/DelugeFirmware/releases/download/release_1_1_1/deluge-community-release-1_1_1.zip">Download c1.1</a>
 
         <h1>Beta - 1.2 (Chopin)</h1>
         <p>
             The 1.2 beta is available below. The beta is approaching release but may still have bugs, please open an issue <a href="https://github.com/SynthstromAudible/DelugeFirmware/issues/new?assignees=&labels=&projects=&template=bug_report.yml">on github</a> if you find one!
         </p>
 
-        <a class="btn" target="_blank" href="https://github.com/SynthstromAudible/DelugeFirmware/releases/download/beta/beta.zip">Download todays beta build</a>
+        <a class="btn" target="_blank" href="https://github.com/SynthstromAudible/DelugeFirmware/releases/download/beta/beta.zip">Download todays beta</a>
+        <a class="btn" taget="_blank" href="https://github.com/SynthstromAudible/DelugeFirmware/blob/release/1.2/CHANGELOG.md">Changelog current beta</a>
         <a class="btn" target="_blank" href="https://github.com/SynthstromAudible/DelugeFirmware/blob/release/1.2/docs/community_features.md">Up to date documentation</a>
-        <a class="btn" taget="_blank" href="https://github.com/SynthstromAudible/DelugeFirmware/blob/release/1.2/CHANGELOG.md">Changelog</a>
-        <h1>Development firmware</h1>
+        <h1>Development firmware aka alpha aka 1.3</h1>
```

Comment:
```suggestion
        <h1>Development firmware aka nightly aka 1.3</h1>
```

It's really pre alpha, changes are made directly to it